NEWS

```diff
@@ -36,22 +37,22 @@ PHP                                                                        NEWS
 
 - DOM:
   . Added DOMNode::compareDocumentPosition(). (nielsdos)
-  . Implement #53655 (Improve speed of DOMNode::C14N() on large XML documents).
+  . Implemented #53655 (Improve speed of DOMNode::C14N() on large XML documents).
```

Comment:
This still exceeds 80 characters.

Reply:
Sorry. I fixed this and fixed all the lines exceeding 80 chars in both files.